src/lib/consts.ts

```diff
@@ -15,3 +15,11 @@ export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000'
 export const NULL_BYTES32 =
   '0x0000000000000000000000000000000000000000000000000000000000000000'
 export const REFERRAL_RECIPIENT = '0x8C9a3Ec0049F7E873760057a3A8bD163c9878aCE' // mesawallet.eth
+
+export const EMPTY_RECIPIENT = { address: '', percentAllocation: 0 }
+export const DEFAULT_RECIPIENTS = [{ address: '', percentAllocation: 0 }]
+export const DEFAULT_DISTRIBUTOR_FEE = 0
+export const DEFAULT_DISTRIBUTOR_FEE_OPTIONS = [0.01, 0.1, 1]
```

Comment:
where is DEFAULT_DISTRIBUTOR_FEE_OPTIONS being used???
```suggestion
export const DEFAULT_DISTRIBUTOR_FEE_OPTIONS = [0.01, 0.1, 1]
```